crates/bevy_render/src/color/mod.rs

```diff
@@ -1782,6 +1847,100 @@ const fn hex_value(b: u8) -> Result<u8, u8> {
 mod tests {
     use super::*;
 
+    #[test]
+    fn lerp_colors() {
```

Comment:
Should something `color_rgba1.lerp(color_lcha2, 0.5)` be equivalent to `color_rgba1.lerp(color_rgba2, 0.5)` and `color_lcha1.lerp(color_lcha2, 0.5)`?

Reply:
I assume it should, and right now it doesn't. Though I think an easy fix would be to convert the colors to `Rgba` for instance and lerping on that, do you think it would be a good way to do it?

Reply:
I think I didn't understand the issue at first. If we want lerping colors to be accurate, we should be about to lerp between color spaces. I think it needs to be discussed further.